ign_rviz_common/include/ignition/rviz/common/rviz_events.hpp

```diff
@@ -45,19 +45,6 @@ class FixedFrameChanged : public QEvent
   static const QEvent::Type kType = QEvent::Type(startingEventId);
 };
 
-/**
- * @brief Event is generated when the frame list is changed
- */
-class FrameListChanged : public QEvent
```

Comment:
I am not sure if I want to remove this yet. We might be able to send the `FrameListChanged` signal by checking at regular intervals. This method (querying at regular intervals) can also be used to send a `TopicListChanged` signal (topic combo-box in plugins is currently updated on clicking the refresh button)

Reply:
Added back the event.